parts/linux/cloud-init/artifacts/krustlet.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Krustlet
+
+[Service]
+Restart=on-failure
+RestartSec=5s
+EnvironmentFile=/etc/default/kubelet
```

Comment:
do we need to read any env vars from /etc/default/kubelet?

Reply:
seems like KUBELET_NODE_LABELS comes from there?

Reply:
Yep just node labels for now